core/forkid/forkid_test.go

```diff
@@ -136,8 +136,8 @@ func TestCreation(t *testing.T) {
 				{21735000, 1650443255, ID{Hash: checksumToBytes(0x018479d3), Next: 1690889660}}, // First GIP-31 block
 				{29242931, 1690889650, ID{Hash: checksumToBytes(0x018479d3), Next: 1690889660}}, // Last pre-Shanghai block
 				{29242932, 1690889660, ID{Hash: checksumToBytes(0x2efe91ba), Next: 1710181820}}, // First Shanghai block
-				{33101363, 1710181815, ID{Hash: checksumToBytes(0x2efe91ba), Next: 1710181820}}, // Last Shanghai block (approx.)
-				{33101364, 1710181820, ID{Hash: checksumToBytes(0x1384dfc1), Next: 0}},          // First Cancun block (approx)
+				{32880679, 1710181815, ID{Hash: checksumToBytes(0x2efe91ba), Next: 1710181820}}, // Last Shanghai block (approx.)
```

Comment:
Actual Gnosis block 32880679 has timestamp 1710181810, not 1710181815 (it looks like 1 slot was missed around the hard fork and the gap between the 2 blocks was 10 secs instead of 5).

Also, you can remove "approx" from the comment here.